### PR TITLE
Cross-Compiling bitcoinz cli wallet for Win

### PR DESCRIPTION
**Pull request by sdustmcore**
OS: Ubuntu 16.04,  using mingw32-w64 to cross-compiling bitcoinz cli wallet for win 
Please read: cross_compiling_bitcoinz_for_win_using_mingw32-w64.txt

### CODE REVIEW

File: src/snark/Makefile [hunk 1]
```diff
@@ -44,80 +49,80 @@ ifneq ($(NO_SUPERCOP),1)
...
 LIB_SRCS = \
-	src/algebra/curves/alt_bn128/alt_bn128_g1.cpp \
-	src/algebra/curves/alt_bn128/alt_bn128_g2.cpp \
-	src/algebra/curves/alt_bn128/alt_bn128_init.cpp \
-	src/algebra/curves/alt_bn128/alt_bn128_pairing.cpp \
-	src/algebra/curves/alt_bn128/alt_bn128_pp.cpp \
-	src/common/profiling.cpp \
-	src/common/utils.cpp \
-	src/gadgetlib1/constraint_profiling.cpp \
+	libsnark/algebra/curves/alt_bn128/alt_bn128_g1.cpp \
```
Comment (renuzit) [inline src/snark/Makefile:52]: Is there any particular reason why libsnark was moved from `src` to `libsnark` directory?

Comment (sdustmcore) [reply src/snark/Makefile:52]: I checkout the latest commit b73b709,  there are src and libsnark in bitcoinz/src/snark/ directory, I think libsnark is the new version, am i right?  I also check github.com/zcash/zcash/src/snark, there is only libsnark sub-directory

Comment (renuzit) [reply src/snark/Makefile:52]: Interesting, I'll have to research that more. I'm not the maintainer of this project, so I don't know until I dig.

**Comment by ghost** [comment]
Hi, please maintain windows code in your own repo. Zcash doesn't provide windows build. 
After merge we may have a lot of problem with merge latest code in the future.

**Comment by renuzit** [comment]
@sdustmcore Please link me to any repo you may create for windows!

**Comment by sdustmcore** [comment]
Ok,  I'll maintain windows code in https://github.com/sdustmcore/bitcoinz , ForWin branch.

**Pull request closed without merging**